### PR TITLE
German.po Update

### PR DESCRIPTION
**Pull request by NiggoJaecha**
I added translations for the sexpositions and thier describtion. The .po file is from poedit but its kinda strange because it created that german.mo file

### CODE REVIEW

**Comment by pubro** [comment]
About PoEdit:
---
Go to the File->Settings menu and see the Editor tab. Uncheck the box that shocks Сaptain Obvious.

**Comment by pubro** [comment]
I looked your translation, and I doubt that it can be called a translation. This is just a copy phrases on the original language.

For example text from an Indian picture book: 
_"Und das ist " Der umgekehrte Schornsteinfeger ", die letzte der 17 grundlegenden Sexstellungen."_

**Comment by NiggoJaecha** [comment]
Thanks... This was the fist time I used Poedit. I'll change the stuff you told me and delete the Mo.

File: german.po [hunk 1]
```diff
@@ -2545,14 +2571,14 @@ msgstr "Etwas Mystisches... Etwas Verbotenes... Etwas... Menschliches!"
 #: /Game/Breeding/Positions/Cowgirl.Default__Cowgirl_C.SessionData.PositionName
 msgctxt ",63F3737B48B4C916545D1D8159EB762C"
 msgid "Cowgirl"
-msgstr ""
+msgstr "Cowgirl"
```
Comment (pubro) [inline german.po]: **.PositionName**
Ich spreche kein Deutsch, aber ich denke, es kann irgendwie so übersetzt werden: "Reitstellung"?

File: german.po [hunk 2]
```diff
@@ -900,7 +907,7 @@ msgstr "Du wirst nur wilde @MONSTER_RACE@ fangen, die ungefähr die Intelligenz
 #: /Game/Breeding/Positions/ReverseCowgirl.Default__ReverseCowgirl_C.SessionData.PositionName
 msgctxt ",21D4E1714A491D7CAA40E99AA6D354E8"
 msgid "Reverse Cowgirl"
-msgstr ""
+msgstr "Reverse Cowgirl"
```
Comment (pubro) [inline german.po]: "umgekehrt"+"Reitstellung" ?

(Ich kenne keine Deutsche Grammatik)

**Comment by NiggoJaecha** [comment]
As far as I know these Sexpostiones have the thier English name in German as well... And German ppl will perfectly understand what is meant with cowgirl and similar... For those who don't there still is the descibtion which is translated to german

**Comment by pubro** [comment]
I'm sorry, maybe I don't know how many borrowed words and phrases there are in German, but it seems to me that if there is a similar word in the native language, then why not use it.

For exemple what we all know:
What  is "popcorn". I have no doubt that any child will answer this question.
But in Russia, in addition to this word, there is a similar expression describing the food itself: "Vozdushnaya kukuruza" (Air-filled corn).
And we can use both phrases equally.

**Comment by NiggoJaecha** [comment]
Here is the whole german Dialogue Script of Parvati! So that you can see what I did with the ryhmes:
[Parvati Script German.txt](https://github.com/DerelictHelmsman/botn-translations/files/4194512/Parvati.Script.German.txt)
Note that the Script might not resemble the .po 100%. I added some marks and capital letters on each line. It may still contain minor mistakes tho, I'll look for further mistakes as soon as I can test it ingame.

**Pull request closed without merging**